core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSparkMetricsAggTrait.scala

```diff
@@ -29,7 +29,8 @@ trait AppSparkMetricsAggTrait extends AppIndexMapperTrait {
    * @return a single record of AggRawMetricsResult containing all the raw aggregated Spark
    *         metrics
    */
-  def getAggRawMetrics(app: AppBase, index: Int): AggRawMetricsResult = {
+  def getAggRawMetrics(app: AppBase, index: Int, sqlAnalyzer: Option[AppSQLPlanAnalyzer] = None):
```

Comment:
nit: We should update the documentation for this function for the new argument

Reply:
thanks, updated this

Reply:
thanks, added documentation